external-docs/docs/guides/configuration/qube-config/columns/observations.md

```diff
@@ -0,0 +1,92 @@
+# Observation columns
+
+This page discusses what an observation column is, where one should be used, and how one can be defined.
+
+> For a detailed look at an observation column's configuration options, see the [Reference table](#reference) at the bottom of this page.
+
+## What is an observation column?
+
+Observation columns contain the numerical values of observations recorded in the data set, and are the most important component of a CSV-W. In order to be valid, a data cube must include at least one observation column, each of which must have a unit and a measure associated with it. Measures and units can either be defined against the observation column, or can be contained in separate [unit](./units.md) and [measure](./measures.md) columns.
+
+## When to use one
+
+Observation columns contain the observed values of your data, and as such your data set must always contain at least one observation column. The configuration of observation columns in your data set will primarily depend on the [shape of your data](../../../shape-data/index.md). This is discussed in more detail below.
+
+## Basic configuration
+
+For [standard shape](../../../shape-data/standard-shape.md) data sets, where units and measures are contained in their own columns, only the `type` and `data_type` fields can be populated. For [pivoted shape](../../../shape-data/pivoted-shape.md) data sets, the `unit` and `measure` fields can also be configured.
```

Comment:
Can you make sure to mention the reserved names you can use when you configure an observations column by convention?